snowfakery/cli.py

```diff
@@ -51,6 +51,33 @@ def int_string_tuple(ctx, param, value=None):
     return string, number
 
 
+class VersionMessage:
+    def quick_test(self) -> str:
+        from io import StringIO
+
+        yaml = """
+        - object: Status
+          fields:
+            quote: Shiny and new
+        """
+        out = StringIO()
+        generate_data(StringIO(yaml), output_files=[out], output_format="txt")
+        return "Properly installed" if out.getvalue() else "Unknown installation error"
+
+    def __mod__(self, vals) -> str:
```

Comment:
This needs a comment explaining how it works, and a test that will break if `click` switches to a different way of formatting the string.